resources/css/components/prose.css

```diff
@@ -0,0 +1,5 @@
+.prose-base {
+    @apply prose font-sans text-neutral text-pretty max-w-none
+    prose-headings:text-neutral prose-a:text-primary prose-strong:text-neutral prose-blockquote:text-neutral
+    prose-a:prose-headings:no-underline prose-a:prose-headings:text-neutral prose-th:text-left;
+}
```

Comment:
do we need additional support for when content has inline-styling due to clients copy pasting existing content? because inline styling would overwrite the prose. Only solution would be for the client to this by hand while we could fix this by using importants

Reply:
In my opinion we don't need to fix that in de package. If this happens inside your project you can fix it there by removing the inline styling or classes.